bugalink-backend/transactions/urls.py

```diff
@@ -8,5 +8,6 @@
 
 urlpatterns = [
     path("", include(router.urls)),
-    path('users/<int:user_id>/transactions/get_last_transactions/', TransactionViewSet.as_view({'get': 'get_last_transactions'}), name='get_last_transactions')
+    path('users/<int:user_id>/transactions/get_last_transactions/', TransactionViewSet.as_view({'get': 'get_last_transactions'}), name='get_last_transactions'),
+    path('users/<int:user_id>/transactions/get_pending_balance/', TransactionViewSet.as_view({'get': 'get_pending_balance'}), name='get_pending_balance')
```

Comment:
Any of these endpoints follow REST naming conventions (`get_last_transactions`, `get_pending_balance`...). Also, is the concept of pending balance even implemented?